bambi/results.py

```diff
@@ -34,80 +37,77 @@ def summary(self):
         pass
 
 
-class PyMC3Results(ModelResults):
-
+class MCMCResults(ModelResults):
```

Comment:
Do we actually need separate `MCMCResults` and `SampleArray` classes at this point? I don't have any objection to keeping them both, but I'm wondering if we couldn't just consolidate them both into `MCMCResults`. I guess the main benefit of keeping the separation is we could potentially store multiple `SampleArray`s in a single `MCMCResults` (especially given the suggested pattern of `.fit(..., backend=['stan', 'pymc'])`). But I think returning a list of `MCMCResult` objects is probably more intuitive in that situation anyway. Thoughts?